x/stake/types/delegation.go

```diff
@@ -9,6 +9,19 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+// struct that just has a delegator validator pair with no other data. To be used to marshal keys
```

Comment:
Let's be consistent with the punctuation/use of full sentences. i.e. pick one or the other 👍